test/unit/services/oooRequest.test.ts

```
@@ -0,0 +1,175 @@
+    });
+
+    describe("validateOOOCreateRequest", () => {
+
+        beforeEach(async ()=> {
+            validOOORequest = await requestModel.add({
+                from: Date.now(),
+                until: Date.now() + convertDaysToMilliseconds(3),
+                type: REQUEST_TYPE.OOO,
+                reason: testReason,
+            });
+        });
```

ESLint

[error] Do not pass arrow functions to beforeEach()

(mocha/no-mocha-arrows)